WebServer/main.go

```diff
@@ -1,46 +1,61 @@
 package main
- import (
+
+import (
+	"encoding/json"
 	"fmt"
 	"log"
 	"net/http"
-	"encoding/json"
- )
- type RequestData struct {
-	Name string `json:"name"`
-	Age int `json:"age"`
+)
+
+type RequestData struct {
+	Name  string `json:"name"`
+	Age   int    `json:"age"`
 	Email string `son:"age"`
- }
- type ResponseData struct {
+}
+type ResponseData struct {
 	Message string `json:"message"`
-	Status string `json:"status"`
- }
-func main(){
-	http.HandleFunc(`/api`,handleRequest)
+	Status  string `json:"status"`
+}
+
+func main() {
+	http.HandleFunc(`/api`, handleRequest)
+	http.HandleFunc(`/status`, handleStatus)
 	fmt.Println("Server running on port 8080")
-	log.Fatal(http.ListenAndServe(`:8080`,nil))
```

Comment:
API endpoint names should be self explanatory. there is a standard way to name HTTP endpoints. explore!

HTTP method has not explicetily  mentioned here. newer version of net/http package provides a way to do it.